Sources/DeepLearning/Operators.swift

```diff
@@ -44,6 +44,7 @@ public extension Tensor where Scalar: TensorFlowFloatingPoint {
                                     epsilon: epsilon)
         return (value, { v in
             let mean = self.mean(alongAxes: axis)
+            let max = self.max(alongAxes: axis)
```

Comment:
This constant isn't used anywhere in this vjp - is it necessary?

Reply:
Oh I probably got the idea wrong, i will remove it.